### DISCUSSION TITLE
authstore key name change

### DISCUSSION THREAD

**Discussion by ElliotHYLee** [pocketbase/pocketbase#3231] [Q&A] [answered]
PB stores the auth data in the localstorage. 

Is it possible to change the default key name "pocketbase_auth" to whatever string?

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, but you have to explicitly invoke the `LocalAuthStore` constructor:

```js
import PocketBase, { LocalAuthStore } from "pocketbase"

const pb = new PocketBase("http://127.0.0.1:8090", new LocalAuthStore("your_custom_key"));
```

**Comment by drewp** [comment]
This works, but superuser's password login to `/app1/_/ ` seems to go to the localstore key `__pb_superuser_auth__`. This of course breaks the login to app1 when I login to app2 (same domain, different path). 

Could PB default to sticking meta.appName into that key (or even both the __pb_superuser_auth__ and pocketbase_auth keys)?

**Reply by ganigeorgiev** [reply]
I'll think on it during the UI refactoring but for now the general recommendation is to use subdomains instead of subpaths when possible because of the origin isolation that comes with it.
